availability-store/Cargo.toml

```diff
@@ -9,7 +9,7 @@ edition = "2018"
 polkadot-primitives = { path = "../primitives" }
 parking_lot = "0.7.1"
 log = "0.4.6"
-parity-codec = "4.1"
+parity-codec = "4.1.4"
```

Comment:
why the need for a patch version? can't we just bump it in Cargo.lock?

Reply:
`Cargo.lock` is definitely enough, but I thought that explicit versions would not harm here, because the previous versions didn't work.

I have removed the patch versions anyway.